lib/tbb_2020.3/build/windows.inc

```diff
@@ -12,7 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export SHELL = cmd
+ifeq (true,$(USING_RTOOLS))
+  export SHELL = sh.exe
+else
+  export SHELL = cmd
+endif
```

Comment:
Could we do this in the code that calls the TBB makefiles, rather than needing to modify them here? (I don't mind the extra /s later on)


If so, could we provide some alternative variable like `WINDOWS_HAS_SH` or something? Would be useful for non-RTools environments like conda, but we could still set the default for that variable based on `USING_RTOOLS`